packages/paypal-commerce-integration/src/paypal-commerce-types.ts

```diff
@@ -173,14 +173,24 @@ export interface PayPalCommerceButtonsOptions {
     createOrder(): Promise<string>;
     onApprove(
         data: ApproveCallbackPayload,
-        actions: ApproveCallbackActions,
+        actions?: ApproveCallbackActions,
```

Comment:
should this field be optional?

Reply:
Actually I don't think so. As far as I remember, there are several providers what don't use actions in onApprove callback. However it does not mean that the field is not provided by PayPal... I will make it required. 

Thanks for your comment!